outputs.tf

```diff
@@ -1,20 +1,4 @@
-#output "rds_instance_id" {
-#  description = "The RDB instance ID we just created"
-#  value       = "${aws_db_instance.db.id}"
-#}
-#
-#output "rds_instance_address" {
-#  description = "The RDS instance internal address"
-#  value       = "${aws_db_instance.db.address}"
-#}
-#
-#output "rds_instance_endpoint" {
-#  description = "The RDS instance endpoint"
-#  value       = "${aws_db_instance.db.endpoint}"
-#}
-#
-#output "security_group_id" {
-#  description = "The security group id for the RDS"
-#  value       = "${aws_security_group.db.id}"
-#}
-
+output "instance_id" {
+  value       = "${join("", aws_db_instance.db_including_name.*.id)}"
```

Comment:
This is only for one type of instance. So this should be renamed to something like `db_including_name_instance_id` (and update the description accordingly) and you should also add one for the `excluding_name` instance